db/migrate/20220221223739_add_similarity_to_name_search.rb

```diff
@@ -2,6 +2,6 @@ class AddSimilarityToNameSearch < ActiveRecord::Migration
   def change
     enable_extension :pg_trgm
 
-    add_index :supporters, :name, name: :name_search_idx, order: {name: :gin_trgm_ops}
+    add_index :supporters, :name, name: :name_search_idx, using: :gin
```

Comment:
Does this actually work and use the trigrams index? I had thought it would need to use the `order: {name: :gin_trgm_ops}` as well.

Reply:
I fixed that, I confused with something I saw when I researched for the error!